pre_commit/commands/install_uninstall.py

```diff
@@ -42,37 +41,55 @@ def make_executable(filename):
     )
 
 
-def install(runner, overwrite=False, hooks=False):
+def get_hook_path(runner, hook_type):
+    if hook_type == 'pre-commit':
+        hook_path = runner.pre_commit_path
+        legacy_path = runner.pre_commit_legacy_path
+    else:
+        hook_path = runner.pre_push_path
+        legacy_path = runner.pre_push_legacy_path
```

Comment:
might be able to simplify this since legacy path will always be hook_path + '.legacy'